client/src/components/Login/index.jsx

```diff
@@ -17,24 +40,26 @@ export default class extends Component {
             <span className="login__logo-text login__logo-text--white">UN</span>
             <span className="login__logo-text login__logo-text--black">OPS</span>
           </section>
-          <form action="" className="login__form">
+          <form action="" className="login__form" onSubmit={this.handelSubmit}>
```

Comment:
You've put the handleSubmit event for the entire form as well as the submit button. It's an unnecessary duplication.
I prefer that we leave the event on the submit button and remove this one.

Reply:
ok